os/include/tinyara/log_dump/log_dump.h

```diff
@@ -26,13 +26,23 @@
  * Definitions
  ********************************************************************************/
 
- #define LOGDUMP_SAVE_START	"1"
- #define LOGDUMP_SAVE_STOP	"2"
+/* Log Dump Thread information */
+#define LOG_DUMP_NAME        "log_dump"		/* Binary manager thread name */
+#define LOG_DUMP_STACKSIZE   16384			/* Binary manager thread stack size */
+#define LOG_DUMP_PRIORITY    100			/* Binary manager thread priority */
```

Comment:
Could you please change Binary Manager to Log Dump in the comments of the above three lines please.